ams/server/src/main/java/com/netease/arctic/server/dashboard/controller/TableController.java

```diff
@@ -552,6 +553,30 @@ public void getTableBranches(Context ctx) {
     ctx.json(OkResponse.of(amsPageResult));
   }
 
+  /**
+   * cancel the running optimizing process of one certain table.
+   *
+   * @param ctx - context for handling the request and response
+   */
+  public void cancelOptimizingProcess(Context ctx) {
+    String catalog = ctx.pathParam("catalog");
+    String db = ctx.pathParam("db");
+    String table = ctx.pathParam("table");
+    Preconditions.checkArgument(
```

Comment:
From a parameter design perspective, it is best to specify the process ID. This avoids mistakenly canceling an unintended process due to delays between the frontend page and backend data.

Reply:
> From a parameter design perspective, it is best to specify the process ID. This avoids mistakenly canceling an unintended process due to delays between the frontend page and backend data.

Added, details can be found in new committed code